examples/causal_inference/interventional_distribution.ipynb

```diff
@@ -0,0 +1,1561 @@
+{
```

Comment:
<p>What are the true data-generating parameters that are being estimated here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/552/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>